docs/arch/system/state/shielded_resource_machine/proving/proof_aggregation.md

```diff
@@ -1,3 +1,39 @@
 # Proof aggregation
+An aggregation proof attests to the validity of all compliance and logic proofs of an RM transaction. With aggregation, the [raw RM proofs](../proving/index.md) (except the delta proof) are no longer needed, so they can be erased. Reducing thereby verification time and transaction size.
 
-TBD
+## Aggregation inputs
+#### Witness
+* the list of [compliance proofs](../../resource_machine/data_structures/compliance_unit/compliance_proof.juvix.md) of the RM transaction,
+* the list of [logic proofs](../../resource_machine/data_structures/action/resource_logic_proof.juvix.md) of the RM transaction.
+
+#### Instance
+* the [compliance verifying key](../../resource_machine/data_structures/compliance_unit/compliance_unit.juvix.md#compliance-unit),
+* The list of [logic verifying keys](../../resource_machine/data_structures/action/index.juvix.md###logicVerifyingInputs) of the RM transaction,
+* The list of [compliance instances](../../resource_machine/data_structures/compliance_unit/compliance_proof.juvix.md####instance) of the RM transaction,
+* The list of [logic instances](../../resource_machine/data_structures/action/resource_logic_proof.juvix.md####instance) of the RM transaction.
+
+!!! note
+    Passing the raw proofs as witnesses means they are not needed to verify the aggregation proof. This is what allows to remove them from the RM transaction.
+
+!!! note
+    The verifier must be aware of what has been verified by the prover during aggregation. This means the aggregation instance must account for the raw verifying keys and raw instances. However, they do not need to appear explicitly. A _binding_ (and possibly short) commitment suffices.
```

Comment:
This note explains why we include vks and instances in the instance, right?

Can we rephrase "This means the aggregation instance must account ..." to "That is why we include raw verifying keys.."?